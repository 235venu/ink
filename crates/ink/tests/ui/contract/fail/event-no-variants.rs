#[ink::event_definition]
pub enum Event {}

fn main() {}
