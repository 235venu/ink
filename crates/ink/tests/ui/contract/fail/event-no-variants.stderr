error: ink! event enum definitions must have at least one variant
 --> tests/ui/contract/fail/event-no-variants.rs:2:1
  |
2 | pub enum Event {}
  | ^^^^^^^^^^^^^^^^^
